scripts/configuration/ac-build-config.xml

```diff
@@ -2,7 +2,7 @@
 <config>
     <sdkInfo>
         <sdkVersion>2.0.0-SNAPSHOT</sdkVersion>
-        <iosVersion>1.13.2</iosVersion>
+        <iosVersion>2.0.0-189+1f5f7725098b3e661f17369ab2da793d82cae674</iosVersion>
```

Comment:
Is this a SHA ID appended to the version?  It just seems odd that it changed like this between 1.x and 2.x

Reply:
That's normal on internal repo.